web/src/app/components/login-page/login-page.component.ts

```diff
@@ -47,20 +48,17 @@ export class LoginPageComponent implements OnInit {
       });
   }
 
-  saveUser(user, provider) {
+  saveUser(user: User, provider: string) {
     this.userService.saveUser(user)
       .subscribe((userData) => {
         this.saveUserSocialDetails(userData, provider);
       });
   }
 
-  saveUserSocialDetails(userData, provider) {
+  saveUserSocialDetails(userData: object, provider: string) {
     const socialDetails = {};
     socialDetails[provider] = userData;
     this.userService.saveUserSocial(socialDetails)
-      .subscribe((socialData) => {
-        console.log('userSocialData', socialData); // TODO: store after dashboard count implementation.
-      });
+      .subscribe();
```

Comment:
Is there more to come in the `subscribe` for success and failure?

Reply:
yes @eddiejaoude . Will handle it in the next PR as I am working on changing the flow of storing the social user details.

Reply:
ok 👍 . Can you add a `@TODO`

Reply:
Sure Will do 👍